Formula/geode.rb

```diff
@@ -0,0 +1,36 @@
+class Geode < Formula
+  desc "In-memory Data Grid for fast transactional data processing"
+  homepage "https://geode.apache.org"
+  url "http://apache.org/dyn/closer.cgi?filename=incubator/geode/1.0.0-incubating.M2/apache-geode-1.0.0-incubating.M2.tar.gz&action=download"
+  version "1.0.0-incubating.M2"
```

Comment:
Yea, the official version is "1.0.0-incubating.M2".  Next release will be "1.0.0-incubating.M3".  And after that "1.0.0-incubating".  We can drop the incubating tag only after we graduate to a top-level Apache project.